test/land/create_bkg_ens.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -e
+################################################
+# 4. CREATE BACKGROUND ENSEMBLE (LETKFOI)
+################################################
+YYYY=2021
+MM=03
+DD=23
+HH=18
+FILEDATE=$YYYY$MM$DD.${HH}0000
+
+project_binary_dir=$1
+project_source_dir=$2
+
+echo "project_binary_dir: $project_binary_dir"
```

Comment:
rather than all of these `echo` commands, what about doing `set -x` for this script?